functional-tests/src/test/java/com/redhat/che/functional/tests/WorkspaceTest.java

```diff
@@ -23,19 +24,24 @@
 public class WorkspaceTest {
 
   private static final Logger LOG = LoggerFactory.getLogger(TestTestClass.class);
+  // When creating workspace with che-starter, all others are stopped. Therefore it is better to
```

Comment:
This comment should not be true anymore - https://github.com/redhat-developer/che-starter/pull/313
(at least on prod-preview)

Reply:
Good point, I've created issue for that. https://github.com/redhat-developer/che-functional-tests/issues/386